tools/editor/plugins/shader_graph_editor_plugin.cpp

```diff
@@ -1382,7 +1386,8 @@ ToolButton *ShaderGraphView::make_editor(String text,GraphNode* gn,int p_id,int
 		t.instance();
 		t->create_from_image(icon_color);
 		edit->set_icon(t);
-		break;
+	} break;
```

Comment:
@akien-mga This doesn't change the logic, and it's necessary in order to add the default case because a variable is defined in the block (see this stackoverflow answer http://stackoverflow.com/questions/92396/why-cant-variables-be-declared-in-a-switch-statement)